_conferences/droidcon-vn-2018.md

```diff
@@ -5,4 +5,8 @@ location: Ho Chi Minh City, VN
 
 date_start: 2018-04-07
 date_end:   2018-04-07
+
+cfp_start: 2018-01-01
+cfp_end:   2018-01-03
```

Comment:
It ends tomorrow?

Reply:
its 1st of march

Reply:
It should be 2018-03-01 then. These are ISO 8601 dates.